core/modelmigration/pacage_test.go

```diff
@@ -30,6 +30,11 @@ func (*ImportTest) TestImports(c *gc.C) {
 	// This package should only depend on other core packages.
 	// If this test fails with a non-core package, please check the dependencies.
 	c.Assert(found, jc.SameContents, []string{
+		"core/credential",
 		"core/database",
+		"core/life",
+		"core/model",
+		"core/user",
+		"internal/uuid",
```

Comment:
Technically shouldn't allow this.

Reply:
If we outsourced to `google/uuid` then this goes away.